solr/core/src/test/org/apache/solr/update/AddBlockUpdateTest.java

```diff
@@ -693,7 +668,6 @@ private void indexSolrInputDocumentsDirectly(List<SolrInputDocument> docs) throw
       h.getCore().getUpdateHandler().addDoc(updateCmd);
       updateCmd.clear();
     }
-    assertU(commit());
```

Comment:
this part was fine, especially for nested docs since we want there to be a higher likelihood of more segments since if there's a bug pertaining to not adding docs in a block, then more segments will help prove this out.